### DISCUSSION TITLE
Приклади створення бота в Slack

### DISCUSSION THREAD

**Discussion by Andygol** [k3ilona/publisher-slack-bot#7] [Ideas]
- https://www.technource.com/blog/how-to-create-a-slackbot-using-golang/
- https://programmingpercy.tech/blog/develop-a-slack-bot-using-golang/

**Comment by vit-um** [comment]
Дякую за посилання, додав їх в документацію та зробив PR, прошу апрувити.

**Comment by vit-um** [comment]
Потрібно змінити структуру репо з ботом: 
https://www.digitalocean.com/community/tutorials/how-to-distribute-go-modules
Команда ініціалізації проекту виглядає наступним чином
`go mod init github.com/your_github_username/pubmodule`
таким чином тека модулю та головний файл мають бути розташовані в корені репозиторію.

Саме така реалізація бота дозволить встановлювати залежності версій та вітдалений запуск бота з репо інструментами go.
Я поки не знаю де та як нам це згодиться, але kbot  нас вчили саме так створювати.  Отже часу на вивчення обмаль, дотримуємось бест практіс від Дениса.
